packages/p2p-v2/src/components/MobileCloseHeader/MobileCloseHeader.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './MobileCloseHeader.scss';
+
+const MobileCloseHeader = () => {
+    return (
+        <div className='p2p-v2-mobile-close-header'>
+            Deriv P2P
+            <button className='p2p-v2-mobile-close-header--icon' onClick={() => window.history.back()}>
+                X
+            </button>
+        </div>
+    );
+};
+
```

Comment:
P2P Close header for mobile view, icon not used temporarily until `@deriv/quill-icons` has the icon:
<img width="1819" alt="Screenshot 2024-01-11 at 3 15 30 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/054abb66-ed1f-452c-90c7-7302828995b0">

Reply:
Updated:
<img width="1819" alt="Screenshot 2024-01-11 at 3 39 34 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/8804c501-5a2d-415e-a324-155c380fbdaf">